packages/react-codemirror/src/lang-cypher/lintWorker.ts

```diff
@@ -1,14 +1,28 @@
-import { validateSemantics } from '@neo4j-cypher/language-support';
+import {
+  DbSchema,
+  lintCypherQuery as _lintCypherQuery,
+  _internalFeatureFlags,
+} from '@neo4j-cypher/language-support';
 import workerpool from 'workerpool';
 
-workerpool.worker({ validateSemantics });
+function lintCypherQuery(
+  query: string,
+  dbSchema: DbSchema,
+  featureFlags: { consoleCommands?: boolean } = {},
+) {
+  // We allow to override the consoleCommands feature flag
+  if (featureFlags.consoleCommands !== undefined) {
+    _internalFeatureFlags.consoleCommands = featureFlags.consoleCommands;
+  }
```

Comment:
Before the internal feature flags (which are a global variable) only affected things that were run in the same thread than the editor (namely the syntax errors). Now we are moving the syntax errors down to the semantic analysis worker, so this change requires us to set that global variable there as well.

There's been talks for reshaping the way we package the language support into a class, but until then, this is the way forward.